packages/chain-adapters/src/evm/base/BaseChainAdapter.ts

```diff
@@ -84,9 +84,9 @@ export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.BaseMainnet> {
     input: GetFeeDataInput<KnownChainIds.BaseMainnet>,
   ): Promise<FeeDataEstimate<KnownChainIds.BaseMainnet>> {
     try {
-      const req = await this.buildEstimateGasRequest(input)
+      const req = this.buildEstimateGasRequest(input)
```

Comment:
Stylistic only: we could rename this to `estimateGasBody` to be more specific.

Reply:
f651bd087e8e1609080ede066a942fd63c4cb300